### PR TITLE
interface: add locale-control from ubuntu-core-security

### PR DESCRIPTION
**Pull request by jdstrand**
Another with no seccomp.

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

File: interfaces/builtin/locale_control.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import (
+	"github.com/ubuntu-core/snappy/interfaces"
+)
+
+// http://bazaar.launchpad.net/~ubuntu-security/ubuntu-core-security/trunk/view/head:/data/apparmor/policygroups/ubuntu-core/16.04/locale-control
+const localeControlConnectedPlugAppArmor = `
+# Description: Can manage locales directly separate from 'config ubuntu-core'.
+# Usage: reserved
+
+# TODO: this won't work until snappy exposes this configurability
```
Comment (zyga) [inline interfaces/builtin/locale_control.go]: Can you explain why this won't work?

**Comment by zyga** [comment]
whitelist this please

**Comment by zyga** [comment]
Same comment as on #721 (``all.go``)

File: interfaces/builtin/locale_control.go (hunk 1 above)
Comment (jdstrand) [inline interfaces/builtin/locale_control.go]: snappy needs to make /etc/default/locale writable.

**Pull request merged**